index.html

```diff
@@ -15,15 +15,10 @@ <h3>
 
   <div class="alert alert-warning">
     <h3 class="alert-heading">
-      We are temporarily suspending in-person hack nights
+      We have resumed in-person hack nights for those who are able.
     </h3>
-
-    <p>
-      In light of the ongoing concerns around COVID-19 we have shifted to
-      <strong>virtual hack nights</strong> until September 1, 2021.
-    </p>
     <p>
-      Please join us online Tuesday nights at 7 p.m. Here is how to get started:
+      Please join us online Tuesday nights at 7 p.m. at Cambridge Innovation Center (4th Floor). Here is how to get started:
```

Comment:
Can you update this to be something like:
> "New members can join us online Tuesday nights at 7 p.m.! Existing members can also attend in-person at CIC (5th Floor Outdoor Patio) by talking to their project lead.

Otherwise this looks good to me!

Reply:
@mzagaja take a look with the updates.